src/utils.ts

```diff
@@ -113,7 +114,11 @@ function getSetting<T extends keyof Settings>(name: T): Settings[T] {
     }
 
     try {
-        return (sessionStorage.getItem(name) as Settings[T]) || DEFAULT_USER_SETTINGS[name];
+        if (name === LOCAL_STORAGE_SETTING.theme) {
+            return (sessionStorage.getItem(name) as Settings[T]) || DEFAULT_USER_SETTINGS[name];
```

Comment:
Looks like this logic should be inverted.
Currently I see what theme stored in sessionStorage, but should be saved in LocalStorage

Reply:
yes, you right.